packages/api/src/utils/client/httpClient.ts

```diff
@@ -270,12 +278,13 @@ export class HttpClient implements IHttpClient {
     args: E["args"],
     init: ApiRequestInitRequired
   ): Promise<ApiResponse<E>> {
-    const {retries, retryDelay, signal} = init;
+    const {requestWireFormat, retries, retryDelay, signal} = init;
     const routeId = definition.operationId;
+    const requestMethod = requestWireFormat === WireFormat.ssz ? this.requestFallbackToJson : this._request;
```

Comment:
should this be factored into a private method? something like
```ts
private getRequestMethod(requestFormat: WireFormat): typeof this._request {
  return requestFormat === WireFormat.ssz ?
    this.requestFallbackToJson.bind(this) :
    this._request.bind(this);
}
```

Reply:
Good call, it's definitley a lot cleaner and less duplication on ssz check